src/app/apps/rup/plantillas-rup/plantillas-rup.component.ts

```diff
@@ -96,20 +99,22 @@ export class PlantillasRUPComponent implements OnInit {
 
         let query = {
             search: this.searchTerm,
-            semanticTag: ['procedimiento', 'elemento de registro']
+            semanticTag: ['procedimiento', 'elemento de registro', 'régimen/tratamiento']
 
         };
 
+
         this.snomedService.get(query).subscribe((resultado: ISnomedConcept[]) => {
             this.procedimientos = resultado;
         });
     }
 
 
     cargarPlantillas(procedimiento) {
+
         this.procedimiento = procedimiento;
         this.subject.next([]);
-        this.sp.get(procedimiento.conceptId, true).subscribe(plantillas => {
+        this.sp.get(procedimiento.conceptId, procedimiento.esSolicitud).subscribe(plantillas => {
```

Comment:
Por eso te debe estar andando mal de nuevo. Eliminaste el tercer parámetro que hacía un forzaba la búsqueda.

```suggestion
        this.sp.get(procedimiento.conceptId, procedimiento.esSolicitud, true).subscribe(plantillas => {
```